src/components/OverviewMap/OverviewMap.tsx

```diff
@@ -1,27 +1,48 @@
+import type { Dispatch, SetStateAction } from 'react';
 import React, { memo, useContext, useState, useCallback, useEffect, useMemo } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { useSelector } from 'react-redux';
-import isEqual from 'lodash.isequal';
 import format from 'date-fns/format';
 import subDays from 'date-fns/addDays';
 import L from 'leaflet';
 import { ViewerContainer, themeColor, themeSpacing } from '@amsterdam/asc-ui';
 
+import Map from 'components/Map';
+import PDOKAutoSuggest from 'components/PDOKAutoSuggest';
 import MapContext from 'containers/MapContext/context';
 import { setAddressAction } from 'containers/MapContext/actions';
 import MAP_OPTIONS from 'shared/services/configuration/map-options';
 import configuration from 'shared/services/configuration/configuration';
 import { featureTolocation, formatPDOKResponse } from 'shared/services/map-location';
-import { makeSelectFilterParams, makeSelectActiveFilter } from 'signals/incident-management/selectors';
-import { initialState } from 'signals/incident-management/reducer';
+import { makeSelectFilterParams } from 'signals/incident-management/selectors';
 import useFetch from 'hooks/useFetch';
 import { incidentIcon, markerIcon } from 'shared/services/configuration/map-markers';
-import Map from 'components/Map';
-import PDOKAutoSuggest from 'components/PDOKAutoSuggest';
-import MarkerCluster from './components/MarkerCluster';
-
-import DetailPanel from './components/DetailPanel';
+import type { IncidentSummary } from 'types/incident';
```

Comment:
I would move this definition in the OverviewMap, is not intended to use this interface globally